sql/catalyst/src/main/scala/org/apache/spark/sql/types/DataType.scala

```diff
@@ -456,6 +456,8 @@ object DataType {
           true
         }
 
+      case (_: NullType, _) if storeAssignmentPolicy == ANSI => true
```

Comment:
I think we can write null to any nullable column even if it's strict policy. We can have a followup PR to discuss it further.

Reply:
Okay.